basenode/src/main/java/io/coti/basenode/data/RejectedTransactionData.java

```diff
@@ -0,0 +1,52 @@
+package io.coti.basenode.data;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.coti.basenode.data.interfaces.IPropagatable;
+import io.coti.basenode.data.interfaces.ISignValidatable;
+import io.coti.basenode.data.interfaces.ISignable;
+import lombok.Data;
+
+import java.time.Instant;
+
+@Data
+public class RejectedTransactionData implements IPropagatable, ISignable, ISignValidatable {

```

Comment:
add entire transaction data to support the FN keeping all of it

Reply:
done